src/lib.rs

```diff
@@ -201,52 +161,53 @@ pub async fn load_query_file(world: &World, file_name: &str) -> Result<usize> {
 /// # Errors
 /// Fails if the file can not be loaded
 pub async fn load_cfg_file(world: &World, file_name: &str) -> Result<usize> {
-    info!("Loading configuration from {}", file_name);
+    info!(
+        "Loading configuration from {}",
+        std::path::Path::new(file_name).canonicalize()?.display()
```

Comment:
We should use `tremor_common::file::canonicalize` here instead

Reply:
and since we use `file_name` below perhaps save a version of `canonical_file` and re-use that in all places we display filename?